app/models/payment/PaymentRecord.scala

```diff
@@ -116,8 +126,9 @@ case class VatPaymentRecord(reference: String,
                             createdOn: String,
                             taxType: String) {
 
-  def isValid(currentDateTime: DateTime): Boolean =
-    Try(new DateTime(createdOn).plusDays(DateTimeConstants.DAYS_PER_WEEK).isAfter(currentDateTime)).getOrElse(false)
+  def isValid(currentDateTime: LocalDateTime): Boolean = {
+   Try(LocalDateTime.parse(createdOn, createdOnFormatter).plusDays(7).isAfter(currentDateTime)).getOrElse(false)
```

Comment:
Need to be careful here. In the previous version currentDate was a Joda DateTime; that is a date/time plus an offset. We should modify the payment history service to pass in an OffsetDateTime. Then we can convert the LocalDateTime to an OffsetDateTime with UTC 0. Finally, we can compare this value with the current date.